bin/collect-profiles.sh

```diff
@@ -44,6 +44,17 @@ go tool pprof -symbolize=remote -svg -output mutex.svg "$SOURCE_URL/debug/pprof/
 echo "Disabling mutex profiling"
 curl -X POST "$SOURCE_URL"'/debug/pprof-mutex/?fraction=0'
 
+echo "Enabling block profiling"
```

Comment:
Should this kind of thing also be done in `ipfs diag profile`? I'd hope that this script now doesn't need to be used as frequently given we have a more agnostic tool built in.

Reply:
It definitely could, I'm just not sure what the criteria is for adding things into that command vs. `collect-profiles.sh`. We've always used the latter operationally which is why I added this to it, but it is also trivial to add to `ipfs diag profile`.  

Is the intention for `ipfs diag profile` to replace `collect-profiles.sh`?

Reply:
IMO yes, since it's cross platform and easier for us to version since the command is tied to the binary.